packages/eslint-plugin/src/utils/getPropertyPriority.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+'use strict';
+
+const PSEUDO_CLASS_PRIORITIES = {
```

Comment:
You can just import these values from the `shared` package directly. Making a copy is dangerous as things may go out of sync over time.

Reply:
according to the `getPriority` function, all pseudo classes have priority over at rules except for `@supports`. should that reflect in the sort as well?

Reply:
Yes, all keys should be sorted by priority order in objects!